core/modules/map.rs

```diff
@@ -615,6 +643,195 @@ impl ModuleMap {
     self.pending_dyn_mod_evaluations_pending.get()
   }
 
+  /// Evaluates an already instantiated ES module.
+  ///
+  /// Returns a receiver handle that resolves when module promise resolves.
+  /// Implementors must manually call [`JsRuntime::run_event_loop`] to drive
+  /// module evaluation future.
```

Comment:
This comment needs to be updated

Reply:
Found three instances of this comment, updated all.